### PR TITLE
MGMT-11148:LVMO-feature-support

### PR DESCRIPTION
**Pull request by Mrudraia**
## List all the issues related to this PR

- [X] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [X] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @ @gamli75 @rewantsoni @avishayt
/cc @ 

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by gamli75** [comment]
@Mrudraia you should add the tech preview & unsupported values in https://github.com/openshift/assisted-service/blob/master/internal/featuresupport/support_levels_list.go

File: internal/featuresupport/support_levels_list.go [hunk 1]
```diff
@@ -141,6 +157,10 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDSINGLENODEEXPANSION,
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
 			},
+			{
+				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDLVM,
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
```
Comment (gamli75) [inline internal/featuresupport/support_levels_list.go:162]: this should be FeatureSupportLevelFeaturesItems0SupportLevelDevPreview and please add a new one for 4.12 (copy 4.11 and change it to supported in 4.12)

File: internal/featuresupport/support_levels_list.go [hunk 2]
```diff
@@ -146,6 +162,42 @@ var SupportLevelsList = models.FeatureSupportLevels{
 				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDVIPAUTOALLOC,
 				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
 			},
+			{
+				FeatureID:    models.FeatureSupportLevelFeaturesItems0FeatureIDLVM,
+				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelSupported,
```
Comment (gamli75) [inline internal/featuresupport/support_levels_list.go:167]:
```suggestion
				SupportLevel: models.FeatureSupportLevelFeaturesItems0SupportLevelDevPreview,
```

Comment (Mrudraia) [reply internal/featuresupport/support_levels_list.go]: resolved

**Review by gamli75 — APPROVED** [review]

**Pull request merged**